src/assets/components/ButtonCard.jsx

```diff
@@ -0,0 +1,11 @@
+function ButtonCard({ onClick, buttonText, disabled }) {
+    return (
+
+            <button onClick={onClick} disabled={disabled}>
+                {buttonText}
```

Comment:
Netjes, 1 button element gebruikt. voor de flexibel gebruik van button-actie is aangeraden om volgende te gebruiken:
onClick={() => console.log(text)}
            disabled={disabled}